carina-api/src/main/java/com/qaprosoft/apitools/validation/JsonValidator.java

```diff
@@ -32,62 +37,98 @@
 import com.github.fge.jsonschema.main.JsonSchemaFactory;
 
 public class JsonValidator {
-    private final static Logger LOGGER = Logger.getLogger(JsonValidator.class);
+	private final static Logger LOGGER = Logger.getLogger(JsonValidator.class);
 
-    public static void validateJson(String expectedJson, String actualJson, JSONCompareMode jsonCompareMode) {
-        try {
-            JSONAssert.assertEquals(expectedJson, actualJson, new JsonKeywordsComparator(jsonCompareMode));
-        } catch (JSONException e) {
-            throw new RuntimeException(e);
-        }
-    }
+	public static void validateJson(String expectedJson, String actualJson, JSONCompareMode jsonCompareMode) {
+		try {
+			JSONAssert.assertEquals(expectedJson, actualJson, new JsonKeywordsComparator(jsonCompareMode));
+		} catch (JSONException e) {
+			throw new RuntimeException(e);
```

Comment:
how about create your own carina-api exception and minimize number of errors from Sonar?